<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-PSP68KB" name="com.Tour.qa.testdata.RegistrationTest" tests="34" failures="3" timestamp="2022-04-22T20:37:49 PKT" time="1194.446" errors="0">
  <testcase name="CheckFillData" time="38.071" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckInvalidEmail" time="38.356" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckduplicateEmail" time="37.977" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckinvalidData" time="38.058" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="emptyFormtTest" time="33.664" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="invalidPasswordTest" time="34.685" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="invalidZipCodeTest" time="41.244" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="invalidMobWithLeassDigit" time="36.369" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="invalidMobNumber" time="36.820" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="enterCharacter10InMob" time="39.230" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [The value &#039;qwertyuiop&#039; is invalid.] but found []">
      <![CDATA[java.lang.AssertionError: expected [The value 'qwertyuiop' is invalid.] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.Tour.qa.testdata.RegistrationTest.enterCharacter10InMob(RegistrationTest.java:181)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- enterCharacter10InMob -->
  <system-out/>
  <testcase name="TestLessDigitinMob" time="40.704" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="TestpwwithInt" time="37.603" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="testPWwithString" time="37.624" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="checkFname" time="39.599" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at com.Tour.qa.testdata.RegistrationTest.checkFname(RegistrationTest.java:255)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkFname -->
  <system-out/>
  <testcase name="checkLname" time="40.100" classname="com.Tour.qa.testdata.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at com.Tour.qa.testdata.RegistrationTest.checkLname(RegistrationTest.java:270)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkLname -->
  <system-out/>
  <testcase name="checkEmailnotMatch" time="37.288" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="checkPWnotMatch" time="38.346" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="checkMandatoryFields" time="39.281" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="vehiclesinvalidgTest" time="36.812" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="checkGroupSizeToString" time="38.573" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="HomePageTitleTest" time="30.498" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="CheckBanner" time="27.286" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateGroupText1" time="31.213" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateGroupSize" time="27.032" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateChoosFromMenu" time="31.913" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateChoosDate" time="33.906" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="HeadingValidate1" time="29.983" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateLoginButton" time="35.197" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateForgot" time="29.876" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading2" time="34.975" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading3" time="28.538" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading4" time="28.418" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="ValidateHeading5" time="33.104" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
  <testcase name="validateService" time="32.103" classname="com.Tour.qa.testdata.RegistrationTest"/>
  <system-out/>
</testsuite> <!-- com.Tour.qa.testdata.RegistrationTest -->
